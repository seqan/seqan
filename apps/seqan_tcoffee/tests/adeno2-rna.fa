>gi|9626553|ref|NC_001454.1| Human adenovirus F, firsU 280 base pairs
CAUCAUCAAUAAUAUACCUUAAAACUGGAAACGAGCCAAUAUGAUAAUGAGGGAGGAGGGACUAGGGGUG
GUGUAAGGUGACGUAGAGGCGGGCGGGGUGGGAAAGGGUGGAGGCGGAUGACGUGUGGGGUCGGAGGACG
GGCGCGGUGCGGCGGAAGUGACGGAAAAUCUGGUGUAUUGGGCGGGUUUUUGUAACUUUUGGCCAUUUUG
GCGCGAAAACUGAGUAAUGAGGACGUGGGACGAACUUUGGACUUUUGUGUUUAUGGAGGAAAAACUGCUG
AUUAUUACUGAACUUUGGCCCAUGACGAACCGGUUUUUCUACGUGGCAGUGCCACGAGACGGCUCAAAGU
>gi|9626158|ref|NC_001405.1| Human adenovirus C, firsU 280 base pairs
CAUCAUCAUAAUAUACCUUAUUUUGGAUUGAAGCCAAUAUGAUAAUGAGGGGGUGGAGUUUGUGACGUGG
CGCGGGGCGUGGGAACGGGGCGGGUGACGUAGUAGUGUGGCGGAAGUGUGAUGUUGCAAGUGUGGCGGAA
CACAUGUAAGCGCCGGAUGUGGUAAAAGUGACGUUUUUGGUGUGCGCCGGUGUAUACGGGAAGUGACAAU
UUUCGCGCGGUUUUAGGCGGAUGUUGUAGUAAAUUUGGGCGUAACCAAGUAAUGUUUGGCCAUUUUCGCG
GGAAAACUGAAUAAGAGGAAGUGAAAUCUGAAUAAUUCUGUGUUACUCAUAGCGCGUAAUAUUUGUCUAG
