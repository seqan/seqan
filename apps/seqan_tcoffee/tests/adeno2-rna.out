>gi|9626553|ref|NC_001454.1| Human adenovirus F, firsU 280 base pairs
CAUCAUCAAUAAUAUACCUUAAAACUGGAAACGAGCCAAUAUGAUAAUGAGGGAGGAGGG
ACUAGGGGUGGUGUAAGGUGACGUAGAGGCGGGCGGGGUGGGAAAGGGUGGAGGCGGAUG
ACGU-GUGGGGUCGGAGGACGGGCG-------CGGUGCGGCGG-----------------
----------AAGUGACG-------------------------------GAAAAUCU---
-GGUGUAUUGGGCGG--GUUUUUGUAACU--------------------UUUGGCCAUUU
UGGCGCGAAAACUGAGUAAUGAGGACGUGGGACGAACUUUGGACUUUUGUGUUUAUGGAG
GAAAAACUGCUGAUUAUUACUGAACUUUGGCCCAUGACGAACCGGUUUUUCUACGUGGCA
GUGCCACGAGACGGCUCAAAGU
>gi|9626158|ref|NC_001405.1| Human adenovirus C, firsU 280 base pairs
CAUCAUC-AUAAUAUACCUU-AUUUUGGAUUGAAGCCAAUAUGAUAAUGAG---------
----GGGGUGGAGU-UUGUGACGU---GGCGCGGGGCGUGGGAACGG-----GGCGGGUG
ACGUAGUAGUGU-GGCGGAAGUGUGAUGUUGCAAGUGUGGCGGAACACAUGUAAGCGCCG
GAUGUGGUAAAAGUGACGUUUUUGGUGUGCGCCGGUGUAUACGGGAAGUGACAAUUUUCG
CGCGGUUUUAGGCGGAUGUUGUAGUAAAUUUGGGCGUAACCAAGUAAUGUUUGGCCAUUU
UCGCGGGAAAACUGAAUAA-GAGGAAGU-----GAAAUCUG-------------------
--AAUAAUUCUG--UGUUACUCA----UAGCGCGUAA------UAUUUGUCUAG------
----------------------
